Moodis/Moodis/Program.cs

```diff
@@ -19,5 +19,9 @@ static void Main()
             Application.SetCompatibleTextRenderingDefault(false);
             Application.Run(new LoginForm());
         }
+        public static void close()
+        {
+            Application.Exit();
```

Comment:
Shouldn't `Application.Exit()` work everywhere else, where you used `Program.close()`? If yes, then `Program.close` method is unnecessary